ee/codegen/src/generators/nodes/bases/base.ts

```diff
@@ -609,6 +609,20 @@ export abstract class BaseNode<
       nodeClass.add(statement)
     );
 
+    nodeClass.add(
+      python.field({
+        name: "label",
+        initializer: python.TypeInstantiation.str(this.nodeContext.getNodeLabel()),
+      })
+    );
+
+    nodeClass.add(
+      python.field({
+        name: "node_id",
+        initializer: python.TypeInstantiation.uuid(this.nodeData.id),
+      })
+    );
```

Comment:
we can minimize the diffs in this PR by moving these two lines above the `this.getNodeDisplayClassBodyStatements().forEach` and then regenerating snapshots. Let's do that to make this easier to review